user-guide/Advanced_Functionality/Swarming/SwarmingBookings.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: SwarmingBookings
+---
+
+# Swarming bookings
+
+Since 10.4.4, it is possible to swarm bookings. To do so, you need the *Swarming* soft-launch option. 
+From 10.5.1 onwards, the feature is placed behind a different soft-launch option, namely *BookingSwarming*.
+
+Swarming bookings can be done by sending a *SwarmingRequestMessage*, to which you add the new hosting agent ID and booking ID. 
+This can be done as follows:
+
+```csharp
+using System;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Swarming;
+
+public class Script
+{
+    public void Run(Engine engine)
+    {
+        var newHostingAgent = 123;
+        var bookingId = Guid.Parse("...");
+
+        var request = SwarmingRequestMessage.ForBooking(newHostingAgent, bookingId);
```

Comment:
I would make this an example of a regular bulk call (ForBookings), there is not much use-case for swarming a single booking, and you can always swarm a single booking with the bulk call if needed.